plugins/version.txt

```diff
@@ -1 +1 @@
-2
+4
```

Comment:
Seems like by mistake you ran the script twice.

Reply:
I bet this will go wrong someday. We really shouln't have gone down this path.

Reply:
what is the significance of this plugins/version.txt ? 
I followed readme steps [1,2] https://github.com/fabric8io/openshift-jenkins-s2i-config#update-plugins